### ISSUE TITLE
scss format not work.

### ISSUE THREAD

**Issue by hezhongfeng** [vuejs/vetur#997]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Win10
- Vetur version:0.14.2
- VS Code version: 1.29.1

## Problem

hello , I have a question about  format scss
I have Use 0.14.2 and set the prettier ,and it‘s also the format config.
```
"vetur.format.defaultFormatter.scss": "prettier",
```
I've tried all kinds of bottom code, the format not work.
```
<style lang="scss">
.biz-editor {border: 1px solid #e7e7eb}
</style>

<style lang="scss">
.biz-editor {
border: 1px solid #e7e7eb}
</style>

<style lang="scss">
.biz-editor {
border: 1px solid #e7e7eb
}
</style>

<style lang="scss">
.biz-editor {
border: 1px solid #e7e7eb;
}
</style>
```

**Comment by illuSioN4ng** [comment]
+1
macOS Mojave also has this problem in less formate

**Comment by octref** [comment]
Please fork https://github.com/octref/veturpack and provide a repro.

**Comment by octref** [comment]
Fixed in #998.

**Comment by octref** [comment]
Try 0.14.3.
